ucp/_libs/ucx_api.pyx

```diff
@@ -0,0 +1,490 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+# cython: language_level=3
+
+import socket
+from libc.stdio cimport FILE, fflush, fclose
+from libc.stdlib cimport free
+from libc.string cimport memset
+from libc.stdint cimport uintptr_t
```

Comment:
As it is used below and I don't think it gets imported by default (though could be wrong here).

```suggestion
from libc.stdint cimport uint16_t, uintptr_t
```

Reply:
It's imported in https://github.com/madsbk/ucx-py/blob/cython_refactoring/ucp/_libs/ucx_api_dep.pxd